docs/pages/build-reference/infrastructure.md

```diff
@@ -5,7 +5,11 @@ sidebar_title: Server infrastructure
 
 import { Collapsible } from '~/ui/components/Collapsible';
 
-This document describes the current build infrastructure as of May 13, 2022. It is likely to change over time, and this document will be updated.
+This document describes the current build infrastructure as of June 27, 2022. It is likely to change over time, and this document will be updated.
+
+## Worker IP addresses
+
+An up-to-date list of worker IP addresses is published [here](https://expo.dev/eas-build-worker-ips.txt).
```

Comment:
```suggestion
Here is the [up-to-date list of worker IP addresses](https://expo.dev/eas-build-worker-ips.txt).
```
Small change to adhere to [current doc style guidelines](https://www.notion.so/expo/Expo-Docs-writing-styleguide-c05e8a2e4a184decbb337d745cdaf7af#57a89e3b4ce44763891876b4589788a9)